monitoring/rid_qualifier/host/config.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from monitoring.monitorlib import auth_validation
+
+
+ENV_KEY_PREFIX = 'MOCK_HOST'
+ENV_KEY_PUBLIC_KEY = '{}_PUBLIC_KEY'.format(ENV_KEY_PREFIX)
```

Comment:
I think it's safe to remove `*PUBLIC_KEY`, `*TOKEN_AUDIENCE`, `*BASE_URL`, and `*DSS` from this configuration as we won't use them, even in the future, for this tool.  We should also remove them from run_locally.sh.

Reply:
Done.